core/src/main/java/org/elasticsearch/indices/breaker/HierarchyCircuitBreakerService.java

```diff
@@ -64,10 +65,17 @@
     public static final Setting<CircuitBreaker.Type> REQUEST_CIRCUIT_BREAKER_TYPE_SETTING =
         new Setting<>("indices.breaker.request.type", "memory", CircuitBreaker.Type::parseValue, Property.NodeScope);
 
-
+    public static final Setting<ByteSizeValue> IN_FLIGHT_CIRCUIT_BREAKER_LIMIT_SETTING =
+        Setting.byteSizeSetting("net.breaker.inflight_requests.limit",
```

Comment:
can we use `network.` as a prefix? also please name the setting IN_FLIGHT_REQUESTS?

@dakrone what do you feel should be the default? Do we want to leave it unbound and let a parent circuit breaker kick in? Also since this is the first non-indices related breaker, do you think we should rename all other settings to have INDICES_ prefix in their name?